static/js/reducers/collections.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import { GET, INITIAL_STATE } from "redux-hammock/constants";
+
+import * as api from "../lib/api";
+import type { Collection } from "../flow/collectionTypes";
+
+export const collectionsListEndpoint = {
+  name:              "collectionsList",
+  verbs:             [GET],
+  initialState:      { ...INITIAL_STATE, data: [] },
+  getFunc:           () => api.getCollections()
+};
+
+export const collectionsEndpoint = {
+  name: "collections",
+  verbs: [GET],
+  initialState: { ...INITIAL_STATE, data: new Map() },
+  getFunc: (collectionKey: string): Promise<Collection> => api.getCollection(collectionKey)
```

Comment:
you're setting a `new Map` to be `data` in the `initialState` - if you want to get the results of the `GET` request to be values in that `Map` you will probably want to use a `getSuccessHandler`, which is basically a little callback that receives `payload, data` and returns the updated `data`. for an example you can look at the `postsEndpoint` object we use in open-discussions (`static/js/reducers/posts.js`).

Reply:
although I just noticed this endpoint isn't yet being used anywhere, so maybe it doesn't matter yet?

Reply:
I copied some code from another PR by Gavin since we were both working on related collection functionality, looks like in this case I copied a function I didn't need for this PR.

Reply:
I will remove it

Reply:
gotcha, sounds good